cmd/podman/containers/run.go

```diff
@@ -71,12 +71,13 @@ func runFlags(cmd *cobra.Command) {
 	flags.UintVar(&runOpts.PreserveFDs, "preserve-fds", 0, "Pass a number of additional file descriptors into the container")
 	_ = cmd.RegisterFlagCompletionFunc(preserveFdsFlagName, completion.AutocompleteNone)
 
-	flags.BoolVarP(&runOpts.Detach, "detach", "d", false, "Run container in background and print container ID")
+	flags.BoolVarP(&runOpts.Detach, "detach", "d", false, "Run a container in background and print container ID")
```

Comment:
Should be a different change/commit.  Note that more articles may be needed to turn it into a correct sentence.

Reply:
Done reverted the change for now.